.prettierrc

```diff
@@ -1,5 +1,4 @@
 {
-  "singleQuote": false,
-  "semi": true,
-  "trailingComma": "none"
-}
+  "singleQuote": true,
```

Comment:
I think it would be helpful to leave as double quote for this PR so it's easier to read the diff if that's OK?

Reply:
eslint-config-algolia enforces singleQuotes.... I guess we can disable that rule.

Reply:
Do it in a subsequent PR maybe?

Reply:
Yeah all good, figured out how to make eslint honor the local prettier config